ci/provisioning/post_provision_config_common_functions.sh

```diff
@@ -182,19 +185,18 @@ rpm_test_version() {
 set_local_repo() {
     local repo_server="$1"
 
-    rm -f "$REPOS_DIR"/daos_ci-"$DISTRO_NAME".repo
-    ln "$REPOS_DIR"/daos_ci-"$DISTRO_NAME"{-"$repo_server",.repo}
+    . /etc/os-release
+
+    rm -f "$REPOS_DIR"/daos_ci-"${ID}${VERSION_ID%%.*}".repo
```

Comment:
(lint) Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"? [SC2140]